modules/nw-metallb-addresspool-cr.adoc

```diff
@@ -45,7 +45,7 @@ Specify each range in CIDR notation or as starting and ending IP addresses separ
 |`spec.autoAssign`
 |`boolean`
 |Optional: Specifies whether MetalLB automatically assigns IP addresses from this pool.
-Specify `false` if you want explicitly request an IP address from this pool with the `metallb.universe.tf/address-pool` annotation.
+If you want to explicitly request an IP address, specify `false` and use the `metallb.universe.tf/loadBalancerIPs` annotation to configure a specific address.
```

Comment:
Get rid of false.
If you want to explicitly request a specific IP address, and use the `metallb.universe.tf/loadBalancerIPs` annotation to configure **the** IP address

autoAssign true or false does not come in play in with **metallb.universe.tf/loadBalancerIPs**: <specific IP> annotations. Only requirement is IP address availability in the ip address pool which ever IP address pool that address belongs.
if autoAssign is false for the IP addresspool,  to obtain any address from that pool service needs to use **metallb.universe.tf/address-pool:**<ip address pool name>

Reply:
@asood-rh so if you use `metallb.universe.tf/address-pool`, you still get an IP from the address pool automatically assigned even if you don't choose a specific one?

Reply:
Hi @palonsoro @asood-rh did we fix on a final text here ?

Reply:
Sorry, I was out. I am now doing some testing.

Reply:
Ok. I could test it and it is enough with setting an address pool in the annotation, it just works.

So the original reason to open this bug was mistaken from my side, so I'll close it. There may be room for improvement in the docs, but it should be handled in different tickets for clarity.

Thanks for your help everybody.